### DISCUSSION TITLE
How to change the oap server pod time zone in K8s

### DISCUSSION THREAD

**Discussion by sdhzlzhk** [apache/skywalking#7778] [General]
Environment
1、oap8.7
2、es7.5
3、k8s1.21
I installed the skywalking V8.7 with Helm3. But I found a issue ,the oap sever pod and es7 pod time zone is UTC, 
how to change to UTC+8 in installing

**Comment by kezhenxu94** [comment]
This is out of the scope of SkyWalking, please learn how to change time zone in Pods, the methods to change any Pods' time zone also work for SkyWalking Pod
